lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "packages": [
-    "packages/*"
+    "packages/**"
```

Comment:
This slipped in, I'm guessing? Can be removed from the PR

Reply:
It needs to be added so the `packages/utilities/gel-foundations-styled-components` is installed, so that eslint doesnt complain with the dependancy see https://ci.news.tools.bbc.co.uk/job/psammead/job/gel-foundations-styled-components/10/display/redirect